### ISSUE TITLE
add mediation unity

### ISSUE THREAD

**Issue by k0il** [kloder-games/godot-admob#148]
**OS target (Android/iOS): android

**Godot version:3.2

how to add implementation in gradle.conf ? i need add mediation for unity

**Comment by k0il** [comment]
not work bro...

**Comment by gustavottc** [comment]
i found here how to implements on Android, but i seems to be very difficult 

https://developers.google.com/admob/android/mediate

how mediate works? can you summarize for me?

**Comment by k0il** [comment]
there is no library in this module
 'com.google.ads.mediation: unity: 3.4.8.0'

So, even though I have added AdMob to mediation. won't work in the game

**Comment by gustavottc** [comment]
yes, the mediation is not implemented on this module, we need to implement, but to help you to do that, i need to you says what mediate is for and how it works

**Comment by k0il** [comment]
Mediation is a feature of AdMob that helps you manage all of the ad sources you use to serve ads to your apps in one place.

https://support.google.com/admob/answer/3063564?hl=en

**Comment by k0il** [comment]
maybe you can add fbads other than unityads

**Comment by gustavottc** [comment]
Why mediation is good? You can earn more money with this?

If we implements this, i think is better implements the FaceBook Ads, because supports every format of ad

**Comment by k0il** [comment]
ok ... please add facebook ads mediation

**Comment by gustavottc** [comment]
@k0il , i can`t work now on ads mediation, because iam upgrading a new admob module focusing on the new version for Android plugins and also on iOS

you can check here: https://github.com/Poing-Studios/GodotAdMob

it's on beta version yet

**Comment by k0il** [comment]
ok thank you
